shared/actions/platform-specific.js.flow

```diff
@@ -1,7 +1,6 @@
 // @flow
 declare export function checkPermissions(): Promise<boolean>
-declare export function setShownPushPrompt(): Promise<void>
-declare export function getShownPushPrompt(): Promise<string>
+declare export function getShownPushPrompt(): Promise<0 | 1>
```

Comment:
`Promise<boolean>` doesn't work?

Reply:
It works, but a `1 | 0` is what's literally returned from the obj-C